File: CS/DxBlazorApplication1/Models/UniversityClass.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace DxBlazorApplication1.Models
{
    public class UniversityClass
    {
        public int Id { get; set; }
        public int? LecturerId { get; set; }
        public string? LecturerIds { get; set; }
        public int Status { get; set; }
        public string? Subject { get; set; }
        public string? Description { get; set; }
        public int Label { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public string? Location { get; set; }
        public bool AllDay { get; set; }
        public int EventType { get; set; }
        public string? RecurrenceInfo { get; set; }
        public string? ReminderInfo { get; set; }
        public string? TimeZoneId { get; set; }
        public string? Grade { get; set; }
    }
}

File: VB/DxBlazorApplication1/Models/UniversityClass.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Linq
Imports System.Text
Imports System.Text.Json.Serialization
Imports System.Threading.Tasks

Namespace DxBlazorApplication1.Models

    Public Class UniversityClass

        Public Property Id As Integer

        Public Property LecturerId As Integer?

        Public Property LecturerIds As String?

        Public Property Status As Integer

        Public Property Subject As String?

        Public Property Description As String?

        Public Property Label As Integer

        Public Property StartTime As DateTime

        Public Property EndTime As DateTime

        Public Property Location As String?

        Public Property AllDay As Boolean

        Public Property EventType As Integer

        Public Property RecurrenceInfo As String?

        Public Property ReminderInfo As String?

        Public Property TimeZoneId As String?

        Public Property Grade As String?
    End Class
End Namespace
